views/homepage.ejs

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
Do we need this file if it is present in frontend branch

Reply:
Since the frontend and backend branches haven't been merged yet, I was trying to test the backend functionality through this homepage. To do this, I added the HTML file to the backend branch. This was necessary because the frontend branch, where the file originally exists, isn't accessible from the backend at this stage of development. Adding it to the backend would have allowed me to properly test the backend routes and functionality without having to wait for the branches to be merged. However, there are certain queries regarding integration that I hope to discuss in out next call.

Reply:
Agreed